website/.env

```diff
@@ -24,3 +24,4 @@ ENABLE_EMAIL_SIGNIN=true
 # inference config
 INFERENCE_SERVER_HOST="http://localhost:8000"
 INFERENCE_SERVER_API_KEY="6969"
+ENABLE_CHAT=true
```

Comment:
it should be disabled by default, otherwise the flag should be `DISABLE_CHAT`

Reply:
This env only used in local or static page (which we don't use any more). So I think it's fine

Reply:
Well, then who ever is deploying the project should know, that if they just delete the variable, chat will remain enabled, they have to explicitly set the value to false for it to be disabled.

Reply:
I think those behaviors should be documented instead.

Reply:
Ok, please don't forget to document it somewhere!